tracing-journald/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "tracing-journald"
+version = "0.1.0"
+authors = ["Benjamin Saunders <ben.e.saunders@gmail.com>"]
+edition = "2018"
+license = "MIT"
+repository = "https://github.com/tokio-rs/tracing"
+homepage = "https://tokio.rs"
+description = "rich journald subscriber for `tracing`"
+
```

Comment:
Before we publish this crate to crates.io, it would be good to add the same metadata as the other tracing crates, add a README, and add links to this crate in the list of `tracing` crates in the main README. It would be fine to merge this as is and do that in a follow-up though.